.github/CODEOWNERS

```diff
@@ -226,17 +226,20 @@
 # ServiceLabel: %OpenAI
 # ServiceOwners: @glharper
 
+# PRLabel: %OpenAI
+sdk/ai/ai-inference-rest @glharper @dargilco
```

Comment:
Where are the service owners and the `ServiceLabel` block?   Right now, there's nobody that will be identified to handle issues.

Reply:
oh.. my bad.  It was in a collapsed section.   All good here.